tests/avifyuv.c

```diff
@@ -150,6 +150,10 @@ int main(int argc, char * argv[])
                         int maxDrift = 0;
 
                         avifImage * image = avifImageCreate(dim, dim, yuvDepth, AVIF_PIXEL_FORMAT_YUV444);
+                        if (!image) {
+                            fprintf(stderr, "Out of memory\n");
```

Comment:
Nit: existing error messages are printed to stdout using printf. See line 258:

```
                                        printf("ERROR: Encountered a drift greater than or equal to MAX_DRIFT(%d): %d\n", MAX_DRIFT, drift);
                                        return 1;
```

See also lines 491 and 509.